process_seq.py

```diff
@@ -416,16 +442,26 @@ def extract_picard_stats(path):
         # Assuming no overlap in stat names
         vcf_st.update(picard_wgs_st)
         vcf_st.update(picard_size_st)
-        vcf_st["Sample Name"] = sample_name
 
         final_stats.append(vcf_st)
 
 log.info(f"Starting writing final stats...")
 
 with open(STATS_OUTPUT_PATH, "w") as statsout:
-    # Assumes all stat entries will have exactly the same headers
-    writer = DictWriter(statsout, final_stats[0].keys(), delimiter="\t")
-    writer.writeheader()
-    writer.writerows(final_stats)
+    # Extract keys (Metric names, e.g. number of snps) from first dictionary
```

Comment:
I'm having a little trouble understanding what's going on here and why switch to the raw csv writer instead of `DictWriter`. `DictWriter` is preferred because 1.) the code is more compact 2.) It's more resilient to changes, and handles column names cleanly (unsure if as written this does)

Reply:
i have no problem using DictWriter, i just couldnt figure out how to format it so that the final-stats.csv had metrics as 1st column, and samples occupying each column therafter (i.e. swapping the cols for rows in your previous code - where the metrics were the first row, and samples occupied each row thereafter)

Reply:
is this not very pythonic?